src/extension/inspire/src/main/java/org/geoserver/inspire/wms/WMSExtendedCapabilitiesProvider.java

```diff
@@ -56,6 +57,8 @@ public List<String> getVendorSpecificCapabilitiesChildDecls(GetCapabilitiesReque
     @Override
     public void registerNamespaces(NamespaceSupport namespaces) {
         ServicesUtils.registerNameSpaces(namespaces);
+        // IGN : We add another xmlns for inspire_dls
```

Comment:
I believe has been copied over from the WCSExtendedCapabilitiesProvider. IGN there stood for Institut Geographic National, it was meant to be an indication of who suggested the change.

I believe the comment is not needed here. I'm not sure about the prefix declaration, is the "inspire_dls" prefix actually used in the WMS output? I only see it in the WFS/WCS outputs.

Reply:
You are right, thanks for spotting it. Reverted.